mne/viz/_brain/surface.py

```diff
@@ -155,15 +175,104 @@ def y(self):
     def z(self):
         return self.coords[:, 2]
 
-    def load_curvature(self):
-        """Load in curvature values from the ?h.curv file."""
-        curv_path = path.join(self.data_path, 'surf', '%s.curv' % self.hemi)
-        self.curv = read_curvature(curv_path, binary=False)
-        self.bin_curv = np.array(self.curv > 0, np.int64)
-        # morphometry (curvature) normalization in order to get gray cortex
-        # TODO: delete self.grey_curv after cortex parameter
-        # will be fully supported
-        color = (self.curv > 0).astype(float)
-        color = 0.5 - (color - 0.5) / 3
-        color = color[:, np.newaxis] * [1, 1, 1]
-        self.grey_curv = color
+
+class _ROI(_Surface):
```

Comment:
I wouldn't subclass surface but rather create a new class. I don't think they'll really need to share many properties.

For now I would just implement the marching cubes + rendering directly in _Brain (maybe in a helper function that takes a volume and ROI values and maps these to visible VTK objects), I don't think a class will really help much here